client/payment-details/summary/index.tsx

```diff
@@ -175,6 +176,51 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 		paymentIntent
 	);
 
+	const relativeTimeRemaining = ( deadlineDate: Moment ): string => {
```

Comment:
This is a nitpick comment. I like to have verbs in the name of the functions. I'd suggest `calculateRelativeTimeRemaining` but maybe it's just me. Feel free to ignore it.

Reply:
I have renamed this :)

Reply:
Marking this as resolved